p3analysis/plot/backend/__init__.py

```diff
@@ -51,3 +54,24 @@ class NavChart(Plot):
 
     def __init__(self, backend):
         super().__init__(backend)
+
+
+def _get_platform_labels(platforms: list[str]) -> dict[str, str]:
+    """
```

Comment:
Missing argument in docstring

Reply:
Fixed in 6471f60f5c0c7a0530a91d6d26bb8b7648419a97.